lighthouse-core/audits/dobetterweb/doctype.js

```diff
@@ -6,6 +6,28 @@
 'use strict';
 
 const Audit = require('../audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on the doctype of a site. This descriptive title is shown to users when the site's doctype is set to HTML. */
```

Comment:
can you `s/of a site/of a page/g`? Or even "the page".
Currently our descriptions dont use 'site' because we're always reporting on just one page.

Reply:
Done!